apps/shooting-clubs/acceptance/features/storage-address.js

```diff
@@ -0,0 +1,187 @@
+  I.see(storageAddressPage.content.postcode);
+});
+
+Scenario('I am taken to the storage-add-another-address step from the address-lookup step', (
+  I,
+  storageAddressPage
+) => {
+  storageAddressPage.selectAddressAndSubmit();
+  I.seeInCurrentUrl(storageAddressPage['another-address-url']);
+});
+
+Scenario('I am taken to the storage-add-another-address step from the manual-address step', (
+  I,
+  storageAddressPage
+) => {
+  I.click(storageAddressPage.links['manual-entry']);
+  storageAddressPage.fillFormAndSubmit(storageAddressPage.fields['address-manual'], storageAddressPage.content.address);
+  I.seeInCurrentUrl(storageAddressPage['another-address-url']);
+});
+
+Scenario('The correct form elements are present on storage-add-another-address step', (
+  I,
+  storageAddressPage
+) => {
+  storageAddressPage.selectAddressAndSubmit();
+  I.seeElements([
+    storageAddressPage.fields.add,
+    storageAddressPage.fields.yes,
+    storageAddressPage.fields.no
+  ]);
+});
+
+Scenario('An error is shown if storage-add-another-address step is not completed', (
+  I,
+  storageAddressPage
+) => {
+  storageAddressPage.selectAddressAndSubmit();
+  I.submitForm();
+  I.seeErrors(storageAddressPage.fields.add);
+});
+
+Scenario('I see the storage-address that was added previously', (
+  I,
+  storageAddressPage
+) => {
+  storageAddressPage.selectAddressAndSubmit();
+  I.see(storageAddressPage.content['display-address']);
+});
+
+Scenario('When I select yes on the add-another-address page, I am taken to the storage-postcode step', (
+  I,
+  storageAddressPage
+) => {
+  storageAddressPage.selectAddressAndSubmit();
+  I.click(storageAddressPage.fields.yes);
+  I.submitForm();
+  I.seeInCurrentUrl(storageAddressPage.url);
+});
+
+Scenario('When I add another address, I can see both addresses on the add-another-address step', (
+  I,
+  storageAddressPage
+) => {
+  storageAddressPage.addMultipleAddresses();
+  I.seeEach([
+    storageAddressPage.content['display-address'],
+    storageAddressPage.content['another-address']
+  ])
+});
+
+Scenario('When I click Delete and there are no addresses to display, I am taken back to the storage-postcode page', (
+  I,
+  storageAddressPage
+) => {
+  storageAddressPage.selectAddressAndSubmit();
+  I.click(storageAddressPage.links.delete);
+  I.seeInCurrentUrl(storageAddressPage.url);
+});
+
+Scenario('When I click Delete, the address is removed from the storage-addresses table', (
+  I,
+  storageAddressPage
+) => {
+  storageAddressPage.addMultipleAddresses();
+  I.click(storageAddressPage.links.delete);
+  I.dontSee(storageAddressPage.content['display-address']);
+});
+
+Scenario('When I select No on add-another-address I am taken to the confirm page', (
+  I,
+  storageAddressPage,
+  confirmPage
+) => {
+  storageAddressPage.selectAddressAndSubmit();
+  I.click(storageAddressPage.fields.no);
+  I.submitForm();
+  I.seeInCurrentUrl(confirmPage.url);
+});
```

Comment:
How come the tests for this functionality are here? Shouldn't it be in common if it is shared across journeys now? We don't need to test the same functionality twice, just that it goes to the right place